### PR TITLE
grafana: explained metrics in log backup row

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/17227
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]
rest lgtm

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -459,6 +459,41 @@ This section provides a detailed description of these key metrics on the **TiKV-
 - Encrypt/decrypt data nanos: The histogram of duration on encrypting/decrypting data each time
 - Read/write encryption meta duration: The time consumed for reading/writing encryption meta files
 
+### Log Backup
+
+- Handle Event Rate: The speed of handling write events
+- Initial Scan Generate Event Throughput: Incremental scanning speed when generating a new listener stream
+- Abnormal Checkpoint TS Lag: Lag of the current checkpoint TS to the present time for each task
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:466]:
```suggestion
- Abnormal Checkpoint TS Lag: The lag of the current checkpoint TS to the present time for each task
```

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -459,6 +459,41 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+- Observed Region Count: The number of Regions currently listened to
+- Errors: The number and type of retryable and non-fatal errors
+- Fatal Errors: The number and type of fatal errors. Usually fatal errors cause the task to be paused.
+- Checkpoint TS of Tasks: Checkpoint TS for each task
+- Flush Duration: The heat map of how long it takes for moving cached data to external storage
+- Initial Scanning Duration: The heat map of how long it takes for incremental scanning when creating a new listening stream
+- Convert Raft Event Duration: The heat map of how long it takes to transform a Raft log entry into backup data after creating a listening stream
+- Command Batch Size: The batch size (within a single Raft group) of the listening Raft command
+- Save to Temp File Duration: The heat map of how long it takes to temporarily store a batch of backup data (spanning several Tasks) into the temporary file area
+- Write to Temp File Duration: The heat map of how long it takes to temporarily store a batch of backup data (from a particular task) into the temporary files area
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:477]:
```suggestion
- Write to Temp File Duration: The heat map of how long it takes to temporarily store a batch of backup data (from a particular task) into the temporary file area
```

File: grafana-tikv-dashboard.md [hunk 3]
```diff
@@ -459,6 +459,41 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+- Internal Message Type: The type of messages received by the actor responsible for the log backup within TiKV
+- Internal Message Handling Duration (P90|P99): The speed of consuming and processing each type of messages
+- Initial Scan RocksDB Throughput: The read traffic generated by RocksDB internal logging during incremental scanning
+- Initial Scan RocksDB Operation: The number of individual operations logged internally by RocksDB during incremental scanning
+- Initial Scanning Trigger Reason: The reason for triggering incremental scanning
+- Region Checkpoint Key Putting: The number of checkpoint operations logged to the PD
+
+> **Note:**
+>
+> The following monitoring items all use TiDB as their data source, but they have some impact on the log backup process. For ease of reference, they are placed in TiKV Details. In addition, TiKV actively pushes progress most of the time, so it is normal for some of the following monitoring items to occasionally not have sampled data.
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:488]:
```suggestion
> The following monitoring metrics all use TiDB nodes as their data source, but they have some impact on the log backup process. Therefore, they are placed in the **TiKV Details** dashboard for ease of reference. TiKV actively pushes progress most of the time, but it is normal for some of the following monitoring metrics to occasionally not have sampled data.
```

File: grafana-tikv-dashboard.md [hunk 4]
```diff
@@ -459,6 +459,41 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+
+- Request Checkpoint Batch Size: The request batch size when the log backup coordinator requests checkpoint information for each TiKV
+- Tick Duration \[P99|P90\]:  The time taken by the tick inside the coordinator
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:491]:
```suggestion
- Tick Duration \[P99|P90\]: The time taken by the tick inside the coordinator
```

File: grafana-tikv-dashboard.md [hunk 5]
```diff
@@ -459,6 +459,41 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+- Region Checkpoint Failure Reason: A record of the reason why a Region checkpoint cannot advance within the coordinator
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:492]:
```suggestion
- Region Checkpoint Failure Reason: The reason why a Region checkpoint cannot advance within the coordinator
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**